src/main/java/com/rackspace/ceres/app/config/CacheConfig.java

```diff
@@ -51,4 +57,17 @@ public CacheConfig(MeterRegistry meterRegistry,
 
     return cache;
   }
+
+  @Bean(name = "reactiveRedisTemplate")
+  public ReactiveRedisTemplate<String, List<String>> reactiveRedisTemplate(
```

Comment:
Why did we creating this bean here?

Reply:
I need a different type of reactiveRedisTemplate i.e. ReactiveRedisTemplate<String, List<String>>. an instance of ReactiveRedisTemplate of type String, String has been already created for other usages.